### ISSUE TITLE
Bug: A link is unclickable 

### ISSUE THREAD

**Issue by serkan-ozel-2015400123** [bounswe/bounswe2018group9#277]
<ion-label position="stacked">
              <b>Password</b>
              <a routerLink="/recover" float-right>Forgot password?</a>
            </ion-label>

This link is unclickable, note that I have changed it from href to routerlink; this does not matter in terms of clickability. However, another reason i.e because it is inside the label may be it is unclickable. If I take it out I can click that. Please resolve this.

**Comment by okyksl** [comment]
@serkan-ozel-2015400123 I think it's clickable but there is no path named **recover**.

**Comment by serkan-ozel-2015400123** [comment]
@okyksl This is not true. Even if there isnt a page there, we should be able to navigage there and get and error or returned by your system to another page. You should test a little to figure out actually

**Comment by okyksl** [comment]
Thanks for the heads up @serkan-ozel-2015400123. It seems like **ion-item** sucks all the **click events**. I will address these issue soon.

Side question: Should I go for a PR where I merge **password-recover**, **href bugs** and **unclickable bug**?

**Comment by serkan-ozel-2015400123** [comment]
Possible. But recover page should take time, so in my profile pr, href bugs are handled. no problem though, I may change master faster than a new pr and you may rebase master afterwards. Still we need recover page :)

**Comment by okyksl** [comment]
@serkan-ozel-2015400123 did we fix this?

**Comment by serkan-ozel-2015400123** [comment]
No we did not. Taking this into comsideration it should be easy. If I do I will open a seperate branch and pr

**Comment by okyksl** [comment]
You can handle this issue if you wish.
